DEVELOPER.md

```diff
@@ -222,6 +222,19 @@ To pull in a merge request you should generally follow the command line instruct
 
 # Build the code and run the tests
 
+## Prerequisites
+You need the `ruby` package manager `rvm` which can be installed using `curl -L https://get.rvm.io | bash -s stable --autolibs=enabled && source ~/.profile` (see the [rvm installation instructions](https://github.com/rvm/rvm) for details) and the `nodejs` package manager `nvm` which can be installed using `wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.26.1/install.sh | bash && source ~/.bashrc` (see the [nvm installation instructions](https://github.com/creationix/nvm) for details).
```

Comment:
@krichter722 
It's better to clarify that Storm actually requires Python / Ruby / Node, not RVM, and NVM.
So we would better to explain below first, 
```
You need the Python, Ruby, Node to build the codes.
Python doesn't have package manager so if you don't have python, you need to install manually. (see the [python download page](https://www.python.org/downloads/) for details)
```
and continue explaining RVM and NVM with clarifying it's for convenience, not requirement. Users can still install Ruby and Node manually.